src/main/kotlin/apply/ui/admin/BaseLayout.kt

```diff
@@ -55,6 +56,7 @@ class BaseLayout(
     private fun createMenuItems(): List<MenuItem> {
         val recruitments = recruitmentService.findAll()
         return listOf(
+            "관리자 계정 관리" of AdministratorsView::class.java,
```

Comment:
c: '관리자' 또는 '관리자 관리'는 어떨까요?

```suggestion
            "관리자 관리" of AdministratorsView::class.java,
```